page.php

```diff
@@ -17,8 +17,27 @@
 
 // Check if this is the style guide page.
 if ( 'style-guide' === $post->post_name ) {
-	$acf_block_types      = acf_get_store( 'block-types' );
-	$context['ts_blocks'] = $acf_block_types->get_data();
+  $styleguide_directory_scans = array(
+    'ts_blocks' => '/views/organisms/blocks',
+    'ts_molecules' => '/views/molecules',
+  );
+  foreach( $styleguide_directory_scans as $context_label => $theme_path ) {
+    $context[$context_label] = [];
+    $directory = get_template_directory() . $theme_path;
+    $element_directories = scandir($directory);
+    if ($element_directories) {
+      foreach ($element_directories as $id) {
+        $styleguide_file = "$directory/$id/styleguide/$id--styleguide-layout.twig";
+        if ( file_exists( $styleguide_file ) ) {
+          $controller_file = "$directory/$id/controller/$id--controller.twig";
+          if ( file_exists( $styleguide_file ) ) {
```

Comment:
We're not longer using the controller directory or --controller on the file, so this needs to be updated.

Reply:
Oop, looks like this was already removed. Nice update!